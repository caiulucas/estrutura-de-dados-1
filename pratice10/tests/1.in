Ex(*$a#.mpl.e:
Adventures in Disneyland
Two people were going to Disneyland when they came to a fork in the road.
The sign read: "Disneyland LEFT."
So they went home.