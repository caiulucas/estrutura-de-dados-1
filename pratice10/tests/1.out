5
a 2
adventures 1
came 1
disneyland 3
e 1
ex 1
fork 1
going 1
home 1
in 2
left 1
mpl 1
people 1
read 1
road 1
sign 1
so 1
the 2
they 2
to 2
two 1
went 1
were 1
when 1
